examples/basic/main.tf

```diff
@@ -1,8 +1,37 @@
-provider "spotinst" {
-  token   = var.spotinst_token
-  account = var.spotinst_account
-}
-
-module "[NAME]" {
+module "ocean-aks-np" {
   source = "../.."
+  # Credentials.
+  spotinst_token   = var.spotinst_token
+  spotinst_account = var.spotinst_account
+  # Configuration
+  name                                   = "clusterName"
+  aks_region                             = "region"
+  controller_cluster_id                  = "controllerId"
+  aks_cluster_name                       = "nodePoolName"
+  aks_infrastructure_resource_group_name = "aksInfrastructureResourceGroupName"
+  aks_resource_group_name                = "resourceGroup"
+  autoscale_is_enabled      = true
+  max_vcpu       = 121
+  max_memory_gib = 120
+  max_scale_down_percentage = 10
```

Comment:
prepend the field-name with autoscale_

Reply:
updated to autoscaler_headroom_automatic_percentage.